docs/styling-with-css/index.md

```diff
@@ -26,7 +26,7 @@ line to the top of `Order.re`:
 ```
 
 The `{||}` string literal is known as a [quoted
-string literal](https://v2.ocaml.org/manual/lex.html#sss:stringliterals), and it
+string literal](https://ocaml.org/manual/5.2/lex.html#sss:stringliterals), and it
```

Comment:
I was a bit confused by this kind of change, because on one hand the PR is reverting to 5.1.1 but on the other it's picking 5.2 urls.

I understand both changes are unrelated?

Reply:
Yeah, I don't think linking to 5.2 URLs here causes any problems because there weren't any changes to the manual in those parts.